src/__tests__/directives.test.ts

```diff
@@ -2,40 +2,44 @@
  * Based on https://github.com/graphql/graphql-js/blob/master/src/execution/__tests__/directives-test.js
  */
 
-import {
-  GraphQLObjectType,
-  GraphQLSchema,
-  GraphQLString,
-  parse
-} from "graphql";
+import { parse } from "graphql";
 import { compileQuery } from "../index";
+import { makeExecutableSchema } from "@graphql-tools/schema";
+import { isCompiledQuery } from "../execution";
 
-const schema = new GraphQLSchema({
-  query: new GraphQLObjectType({
-    name: "TestType",
-    fields: {
-      a: {
-        type: GraphQLString,
-        resolve() {
-          return "a";
-        }
-      },
-      b: {
-        type: GraphQLString,
-        resolve() {
-          return "b";
-        }
-      }
+const testSchema = makeExecutableSchema({
```

Comment:
I would a test confirming the resolvers are not being called.

Reply:
done 👍